content/docs/prerequisites/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Prerequisites"
```

Comment:
Can we also move : http://10.x.x.x:1313/csm-docs/docs/csidriver/#features-and-capabilities
under the same chapter and maybe rename it support matrix ?

The idea is to have one page with everything.

Reply:
@coulof - keeping the scope of these changes narrowed for now but this can be addressed in another iteration.  Thank you for your feedback.